packages/fast-components/src/anchor/anchor.ts

```diff
@@ -0,0 +1,58 @@
+import { attr, FastElement } from "@microsoft/fast-element";
+import { ButtonAppearance } from "../button";
+
+export type AnchorAppearance = ButtonAppearance | "hypertext";
+
+export class Anchor extends FastElement {
+    @attr
+    public appearance: AnchorAppearance = "neutral";
+    public appearanceChanged(
+        oldValue: AnchorAppearance,
+        newValue: AnchorAppearance
+    ): void {
+        if (oldValue !== newValue) {
+            this.classList.add(`${newValue}`);
+            this.classList.remove(`${oldValue}`);
```

Comment:
seems like you could just skip the backtick templating and just do `this.classList.add(newValue)`, etc.

Reply:
updated